arm_controller:
  type: position_controllers/JointTrajectoryController
  joints:
     - shoulder_pan_joint
     - shoulder_lift_joint
     - elbow_joint
     - wrist_1_joint
     - wrist_2_joint
     - wrist_3_joint
  constraints:
      goal_time: 0.6
      stopped_velocity_tolerance: 0.05
      shoulder_pan_joint: {trajectory: 0.1, goal: 0.1}
      shoulder_lift_joint: {trajectory: 0.1, goal: 0.1}
      elbow_joint: {trajectory: 0.1, goal: 0.1}
      wrist_1_joint: {trajectory: 0.1, goal: 0.1}
      wrist_2_joint: {trajectory: 0.1, goal: 0.1}
      wrist_3_joint: {trajectory: 0.1, goal: 0.1}
  stop_trajectory_duration: 0.5
  state_publish_rate:  25
  action_monitor_rate: 10
joint_group_position_controller:
  type: position_controllers/JointGroupPositionController
  joints:
     - shoulder_pan_joint
     - shoulder_lift_joint
     - elbow_joint
     - wrist_1_joint
     - wrist_2_joint
     - wrist_3_joint
     
rh_p12_rn_position:
  type: effort_controllers/JointPositionController
  joint: rh_p12_rn
  pid: {p: 7000.0, i: 0.1, d: 300.0}

rh_r2_position:
  type: effort_controllers/JointPositionController
  joint: rh_r2
  pid: {p: 7000.0, i: 0.1, d: 300.0}

rh_l1_position:
  type: effort_controllers/JointPositionController
  joint: rh_l1
  pid: {p: 7000.0, i: 0.1, d: 300.0}

rh_l2_position:
  type: effort_controllers/JointPositionController
  joint: rh_l2
  pid: {p: 7000.0, i: 0.1, d: 300.0}